### PR TITLE
Feature/update iroha merged

### PR DESCRIPTION
**Pull request by Alexey-N-Chernyshov**
Update iroha bindings. Again and again.

### CODE REVIEW

**Comment by Alexey-N-Chernyshov** [comment]
Need to update iroha library for linux to pass CI.

**Comment by Alexey-N-Chernyshov** [comment]
Probably it is good idea to remove iroha subtree.

**Comment by x3medima17** [comment]
@Alexey-N-Chernyshov, I think we can close this PR

File: src/test/integration/ContractsTest.kt [hunk 1]
```diff
@@ -100,6 +100,7 @@ class ContractsTest {
     fun setup() {
         token = deployHelper.deployBasicCoinSmartContract()
         master = deployHelper.deployMasterSmartContract()
+        // println(master.contractAddress) // TODO remove after eth testnet deployment
```
Comment (Alexey-N-Chernyshov) [inline src/test/integration/ContractsTest.kt]: it is for testing purposes

File: src/test/integration/ContractsTest.kt [hunk 2]
```diff
@@ -122,6 +123,7 @@ class ContractsTest {
                 DefaultBlockParameterName.LATEST
             ).send().balance
         )
+        // sendAddPeer(deployHelper.credentials.address) // TODO remove after eth testnet deployment
```
Comment (Alexey-N-Chernyshov) [inline src/test/integration/ContractsTest.kt]: sap

**Pull request merged**